sunpy/io/cdf.py

```diff
@@ -63,11 +64,18 @@ def read_cdf(fname):
 
             data = cdf.varget(var_key)
             # Get units
-            unit = attrs['UNITS']
```

Comment:
should this be kept? If there are data that have no units (and shouldnt have them) then this makes sense, and then you done have the warnings thrown at you?

Reply:
The line below is the same, but I've just renamed `unit` > `unit_str`?